tools/loader/Loader.h

```diff
@@ -37,10 +38,14 @@ class Loader {
   std::string caffe2NetWeightFilename_;
   /// ONNX model file name.
   std::string onnxModelFilename_;
-  /// Execution engine for compiling and running.
-  ExecutionEngine EE_{};
+  /// Host Manager for running the model.
+  std::unique_ptr<glow::runtime::HostManager> hostManager_;
+  /// Backend used for saving bundle and quantization.
+  glow::Backend *backend_;
```

Comment:
We may need multiple backends here to support heterogeneous partitioning. But this can be done in a separate PR.

Reply:
This backend is just for saving and profiling, which we should make part of the runtime (since we'll need the Partitioner anyway).